### PR TITLE
Determine pidfile location from RabbitMQ version

### PR DESCRIPTION
**Pull request by cdonati**
Xenial's RabbitMQ uses a different pidfile location.

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/3328/

File: AppController/lib/taskqueue.rb [hunk 1]
```diff
@@ -76,11 +80,21 @@ def self.start_rabbitmq
        raise AppScaleException.new(msg)
     end
 
+    pidfile = File.join('/', 'var', 'lib', 'rabbitmq', 'mnesia',
+                        "rabbit@#{Socket.gethostname}.pid")
+    begin
+      rabbitmq_version = self.get_rabbitmq_version
+      if rabbitmq_version[0] <= 3 && rabbitmq_version[1] < 4
```
Comment (jeanleonov) [inline AppController/lib/taskqueue.rb]: It seems that statement won't work as expected for 2.8.* (for example).
`if rabbitmq_version[0] < 3 || (rabbitmq_version[0] == 3 && rabbitmq_version[1] < 4)`

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/3344/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/3344/

**Pull request merged**